src/coreclr/nativeaot/Runtime/amd64/GcProbe.S

```diff
@@ -117,7 +112,7 @@ NESTED_ENTRY RhpGcProbeHijack, _TEXT, NoHandler
         ret
 
 LOCAL_LABEL(WaitForGC):
-        or          ecx, DEFAULT_FRAME_SAVE_FLAGS  + PTFF_SAVE_RAX + PTFF_SAVE_RDX
+        mov         ecx, DEFAULT_FRAME_SAVE_FLAGS  + PTFF_SAVE_RAX + PTFF_SAVE_RDX + PTFF_THREAD_HIJACK
```

Comment:
The `PTFF_THREAD_HIJACK` is the NativeAOT couterpart to the `HijackFrame`.
This is basically a way to mark hijacked frames as leaf/active frames.